app/reducers/ticker.js

```diff
@@ -14,24 +15,34 @@ export const GET_TICKERS = 'GET_TICKERS'
 export const RECIEVE_TICKERS = 'RECIEVE_TICKERS'
 
 // Map for crypto codes to crypto tickers
-const cryptoTickers = {
+const DEFAULT_CRYPTO_UNITS = {
   bitcoin: 'btc',
   litecoin: 'ltc'
 }
 
 // Map for crypto names to crypto tickers
-const cryptoNames = {
+const CRYPTO_NAMES = {
   bitcoin: 'Bitcoin',
   litecoin: 'Litecoin'
 }
 
 // ------------------------------------
 // Actions
 // ------------------------------------
-export function setCurrency(currency) {
-  return {
+export const setCurrency = unit => async (dispatch, getState) => {
+  dispatch({
     type: SET_CURRENCY,
-    currency
+    currency: unit
+  })
+
+  const chain = getState().ticker.crypto
```

Comment:
Consider using  `cryptoSelector` that does that

Reply:
Updated.